annotations/src/main/java/mindustry/annotations/Annotations.java

```diff
@@ -187,18 +187,16 @@
 
     /** A set of two booleans, one specifying server and one specifying client. */
     public enum Loc{
-        /** Method can only be invoked on the client from the server. */
+        /** Server only. */
```

Comment:
This is even less helpful, you're just re-stating the name of the enum. The old comment at least explains where it runs and who can invoke it.

Reply:
Except that it was wrong half of the time since the meanings are completely different depending on which parameter its filling in for.